### PR TITLE
feat: explore page

### PR DESCRIPTION
**Pull request by himadri2110**
- Explore page to display videos
- Added videos to DB
- GET call to fetch videos on initial render
- Made util functions to dynamically fetch video and creator images
- Video Context
- Favicon
- Redirects file

PS: Will add more videos later

### CODE REVIEW

**Review by pritam-kr — APPROVED** [review]
Hey, Everything looks good to me

- clean code
- helper function for thumbnail
- suggestion: you can add a custom view and uploaded date, which help you to sort videos by latest later

**Comment by himadri2110** [comment]
> Hey, Everything looks good to me
> 
> * clean code
> * helper function for thumbnail
> * suggestion: you can add a custom view and uploaded date, which help you to sort videos by latest later

Thanks for the feedback.
I will add date and views count while implementing the sort feature. Thank you.

File: src/components/Navbar/Navbar.css [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+}
+
+.nav-bar .nav-brand .text-grey {
+  color: var(--text-secondary);
+}
+
+.nav-bar .nav-search > *,
+.nav-bar .btn {
+  font-size: 0.9rem;
+}
```
Comment (Varu98) [inline src/components/Navbar/Navbar.css:24]: Good use of CSS selectors

File: src/components/VideoCard/VideoCard.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+}
+
+.video-card .card-heading {
+  padding: 0;
+  font-size: 0.9rem;
+  font-weight: bold;
+  display: -webkit-box;
+  -webkit-line-clamp: 2;
+  -webkit-box-orient: vertical;
+  overflow: hidden;
```
Comment (Varu98) [inline src/components/VideoCard/VideoCard.css:38]: The use of webkit property often generates cross-platform issues, maybe also give a check if it works fine across different browsers

**Review by Varu98 — APPROVED** [review]
Clean Code 👍🏻

File: src/components/VideoCard/VideoCard.css (hunk 2 above)
Comment (himadri2110) [reply src/components/VideoCard/VideoCard.css:38]: Yes, I checked. It works well in other browsers as well. Thank you.

**Pull request merged**